pact/concrete-policies/guard-policy/guard-policy-v1.pact

```diff
@@ -92,8 +113,31 @@
     )
   )
 
-  (defun get-guards:object{guards} (token:object{token-info})
-    (read policy-guards (at 'id token))
+  (defun get-uri-guard:guard (token-id:string)
+    (with-read uri-guards token-id {
+      "uri-guard":= uri-guard
+    }
+    uri-guard
+    )
+  )
+
+  (defun get-guards:object{all_guards} (token:object{token-info})
+    (with-read policy-guards (at 'id token) {
+       'mint-guard:= mint-guard
+      ,'burn-guard:= burn-guard
+      ,'sale-guard:= sale-guard
+      ,'transfer-guard:= transfer-guard
+      }
+      (with-default-read uri-guards (at 'id token)
+        {'uri-guard: GUARD_FAILURE}
```

Comment:
This is to handle "old" tokens that do not have `uri-guard` set, right?

If yes, does it make sense to also add this to `get-uri-guard` function so it fails with the same error rather than with `with-read: row not found`?